OTTheme/ottheme-colors/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ottheme-colors",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

Comment:
please can you revert the version change, we handle this during publish with lerna

Reply:
I reverted the change